commands/management/add_mysql.go

```diff
@@ -268,5 +274,6 @@ func init() {
 	AddMySQLC.Flag("skip-connection-check", "Skip connection check").BoolVar(&AddMySQL.SkipConnectionCheck)
 	AddMySQLC.Flag("tls", "Use TLS to connect to the database").BoolVar(&AddMySQL.TLS)
 	AddMySQLC.Flag("tls-skip-verify", "Skip TLS certificates validation").BoolVar(&AddMySQL.TLSSkipVerify)
+	AddMySQLC.Flag("create-user", "Create user root").Hidden().BoolVar(&AddMySQL.CreateUser)
```

Comment:
flag should be hidden for now

Reply:
It is hidden in help, so I am not sure how you mean it?

Reply:
oh, sorry missed that :+1:

Reply:
No problem.